bdb/info.c

```diff
@@ -1673,19 +1673,22 @@ void bdb_process_user_command(bdb_state_type *bdb_state, char *line, int lline,
         process_add(bdb_state, host);
     } else if (tokcmp(tok, ltok, "hostname") == 0) {
         char *iHost, *oHost;
-        Pthread_mutex_lock(&hostname_lookup_lk);
-        gbl_verbose_hostname_lookup = 1;
         iHost = segtok(line, lline, &st, &ltok);
         if (ltok == 0) {
             logmsg(LOGMSG_USER, "expected nodenum\n");
-            gbl_verbose_hostname_lookup = 0;
-            Pthread_mutex_unlock(&hostname_lookup_lk);
             return;
         }
         oHost = tokdup_hostname(iHost, ltok);
-        logmsg(LOGMSG_USER, "%s:%d got hostname: %s for node: %s\n",__func__, __LINE__, oHost, iHost); 
-        gbl_verbose_hostname_lookup = 0;
-        Pthread_mutex_unlock(&hostname_lookup_lk);
+        logmsgf(LOGMSG_USER, out, "%s\n", oHost);
+    }  else if (tokcmp(tok, ltok, "nodenum") == 0) {
+        char *iHost, *oHost;
+        iHost = segtok(line, lline, &st, &ltok);
+        if (ltok == 0) {
+            logmsg(LOGMSG_USER, "expected hostname\n");
+            return;
+        }
+        oHost = tokdup_hostname(iHost, ltok);
```

Comment:
I think this goes to `node_hostname_str` but maybe you want to go to `host_nodenum_str`?

Reply:
thank you for catching this. i'll fix it to just use tokdup